webview/src/plots/components/customPlots/CustomPlotsWrapper.tsx

```diff
@@ -39,6 +48,14 @@ export const CustomPlotsWrapper: React.FC = () => {
       hasItems={hasItems}
       height={height}
     >
+      {hasMissingCheckpointData && (
+        <div className={styles.plotsSectionMessage}>
```

Comment:
Is there a better place to show a message about needing to select checkpoint experiments.